samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/ShadowContainerSamplePage.WinUI.xaml

```diff
@@ -16,21 +16,23 @@
 		<Color x:Key="UnoPink">#f85977</Color>
 		<Color x:Key="UnoBlue">#159bff</Color>
 		<Color x:Key="UnoGreen">#67e5ad</Color>
+		<Color x:Key="purple">#6858d3</Color>
+		
 
 		<utu:Shadow x:Key="DefaultShadow"
 					BlurRadius="20"
 					OffsetX="10"
 					OffsetY="10"
 					Opacity="0.5"
 					Spread="0"
-					Color="{StaticResource UnoColor}" />
+					Color="{StaticResource UnoPink}" />
```

Comment:
Is the pink a marker color for tests? or just debugging colors?

Reply:
can you comment here to confirm? cause i think this is just for debugging, and shouldnt be here